### ISSUE TITLE
error matching for cargo does not work - fix included

### ISSUE THREAD

**Issue by colin-kiegel** [noseglid/atom-build#114]
I am using atom-build on rust-files. However the built-in regexp don't match the output of the build tool 'cargo'.

Example output of cargo:
> src/lexer/job/state/data.rs:19:5: 19:29 error: trait `Extract` is private

1. Original regexp which does *not* match `'^(?<file>[^\\.]+.rs):(?<line>\\d+):(?<col>\\d+):'`
2. Modified regexp which does match `'(?<file>.+.rs):(?<line>\\d+):(?<col>\\d+):'`

The problematic part is the '^' and '[^\\.]' which obviously don't work as they should (maybe a bug in the regexp-library?)

Note: **linter-rust** is using a very similar regexp, which also works [`'(?<file>.+):(?<line>\\d+):(?<col>\\d+):\\s*(\\d+):(\\d+)\\s+((?<error>error|fatal error)|(?<warning>warning)|(?<info>note)):\\s+(?<message>.+)\n'`](https://github.com/AtomLinter/linter-rust/blob/34ae462db02dc308bf98d104afb0a5447fed65e8/lib/linter-rust.coffee)

**Atom Version**: 1.0.0
**System**: Ubuntu 15.04
**(https://github.com/noseglid/atom-build) package, v0.38.0

### Config

```json
{
  "core": {
    "disabledPackages": [
      "vim-mode"
    ],
    "projectHome": "/home/colin/projects",
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      "*~"
    ],
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  },
  "build": {
    "scrollOnError": true,
    "saveOnBuild": true,
    "minimizedHeight": 0.3,
    "monocleHeight": 0.66,
    "panelVisibility": "Keep Visible"
  }
}
```

### Installed Packages

```coffee
# User
build, v0.38.0
docblockr, v0.7.2
git-log, v0.4.1
keyboard-localization, v1.4.3
language-rust, v0.4.3
linter, v1.1.0
linter-rust, v0.1.0
minimap, v4.10.0
php-twig, v4.0.0
project-manager, v1.15.10
racer, v0.15.2
rust-api-docs-helper, v0.5.1

# Dev
No dev packages
```

**Comment by colin-kiegel** [comment]
I have fixed it here https://github.com/noseglid/atom-build/pull/113/commits

**Comment by noseglid** [comment]
Fixed by #113
